### PR TITLE
Refactor to mvc

### PR DESCRIPTION
**Pull request by hellma314**
start of refactoring to mvc pattern - double check required.

### CODE REVIEW

File: Software/main.py [hunk 1]
```diff
@@ -1,257 +1,203 @@
...
-
-            elif (msg.topic == topic_from_controller):
-                # message received from controller
-                new_number = json_loaded["new_number"]
-                print("Received new ticket number from controller: " +
-                      str(new_number))
-                # Check if new number is already in use
-                for place in list_of_places:
-                    if place.ticket_number == new_number:
+                        # Place was gi  ven up by owner
```
Comment (cboltz) [inline Software/main.py]: Please fix `gi  ven` ;-)

**Comment by cboltz** [comment]
Just wondering - should `list_of_labels_*` really live in model.py? IMHO it would better fit into view.py, and either be set from main.py, or (better, but slightly more work) automagically when updating the values in model.py.

Note: this is not a blocker for this MR, and could be done as a follow-up MR if you like this idea.

File: Software/model/model.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from Software.model import place
+
+
+class Model(object):
+
+    def __init__(self, numberOfPlaces):
+        self.numberOfPlaces = numberOfPlaces;
+        self.list_of_labels_to_display_ticket_number = [];
+        self.list_of_labels_to_display_place_number = [];
+        self.list_of_labels_to_display_in_queue = [];
```
Comment (cboltz) [inline Software/model/model.py]: Please remove the `;` at the end of these lines, and also at `self.initPlaces();` some lines below

File: Software/view/view.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from tkinter import Tk, Label
+
+class View:
+    def __init__(self, numberOfPlaces):
+        self.numberOfPlaces = numberOfPlaces;
```
Comment (cboltz) [inline Software/view/view.py]: Another superfluous `;`, please remove it

File: Software/main.py [hunk 4]
```diff
@@ -1,257 +1,203 @@
...
-from datetime import datetime
+import random
 import statistics
-import sqlite3
+from datetime import datetime
+
+from paho.mqtt import client as mqtt_client
+
+from Software.model.model import Model
+from Software.model.place import PlaceState, Place
```
Comment (cboltz) [inline Software/main.py]: These two should probably be `from model.[...]` (without `Software.`)

File: Software/model/model.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+from Software.model import place
```
Comment (cboltz) [inline Software/model/model.py]: That should probably be `import place` or better `from place import Place` (needs a minor change in the last line)

File: Software/main.py [hunk 6]
```diff
@@ -1,257 +1,203 @@
...
 from config import broker, username, password
+from model import model
```
Comment (cboltz) [inline Software/main.py]: `model` is unused, you can drop that import.

File: Software/main.py (hunk 1 above)
Comment (hellma314) [reply Software/main.py]: fixed

File: Software/model/model.py (hunk 2 above)
Comment (hellma314) [reply Software/model/model.py]: fixed

File: Software/view/view.py (hunk 3 above)
Comment (hellma314) [reply Software/view/view.py]: fixed

File: Software/main.py (hunk 4 above)
Comment (hellma314) [reply Software/main.py]: Ill try to stick to your suggested convention - fixed.

**Pull request merged**